packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -5,14 +5,17 @@
 import 'package:file/file.dart';
 import 'package:file/local.dart';
 import 'package:file/memory.dart';
+import 'package:file/record_replay.dart';
 import 'package:path/path.dart' as path;
 
 import 'common.dart' show throwToolExit;
 import 'context.dart';
+import 'process.dart';
 
 export 'package:file/file.dart';
 export 'package:file/local.dart';
 
+const String _kType = 'file';
```

Comment:
Maybe call this kRecordingType? Since it is not really the type of the file_system. It confused me a little bit...